yaml/com/ibm/PLDM/ChapData.interface.yaml

```diff
@@ -0,0 +1,15 @@
+description: >
+    Implement this interface to pass Chap encrypted string to Host from PLDM.
+
+    com.ibm.PLDM.ChapData object implements this interface.
+
+properties:
+    - name: ChapName
+      type: string
+      description: >
+          ChapName is name associated with each secret chap password.
```

Comment:
"ChapName is name associated with each secret chap password." --> "ChapName is a user selected name associated with each of the chap secret password."